src/Analyser/TypeSpecifier.php

```diff
@@ -235,9 +237,19 @@ public function specifyTypesInCondition(
 							$newContext = $newContext->negate();
 						}
 						$argType = $scope->getType($exprNode->getArgs()[0]->value);
-						if ($argType instanceof StringType) {
+						if ($argType->isString()->yes()) {
+							$accessories = [
+								new AccessoryNonEmptyStringType(),
+							];
```

Comment:
I guess it should not be a problem atm, but wouldn't it make more sense / be safer or more consistent to do something like the following instead?
```suggestion
                        $accessories = [];
						if ($argType->isString()->yes()) {
							$accessories[] = new AccessoryNonEmptyStringType();
```

Reply:
ah sorry, I misread the if indentation :)